expat/lib/expat_external.h

```diff
@@ -88,8 +88,7 @@
 #    ifndef XML_BUILDING_EXPAT
 /* using Expat from an application */
 
-#      if defined(_MSC_EXTENSIONS) && ! defined(__BEOS__)                      \
-          && ! defined(__CYGWIN__)
+#      if defined(_MSC_VER) && ! defined(__BEOS__) && ! defined(__CYGWIN__)
```

Comment:
Does `_MSC_VER` ever occur together with `__BEOS__` or `__CYGWIN` (when using a built expat package)?

Reply:
@dg0yt that's a very valid question that I am afraid I do not know the answer to.